cmd/alertmanager/main.go

```diff
@@ -206,7 +204,7 @@ func main() {
 			cancel()
 			peer.Leave(10 * time.Second)
 		}()
-		go peer.Settle(ctx, *gossipInterval*10)
+		go peer.Settle(ctx, *pushPullInterval)
```

Comment:
If I understand the `Settle()` function correctly, it initially waits for `interval` before starting to check whether the cluster is settled. By increasing this interval (`gossipInterval` -> `pushPullInterval`) marking a cluster as settled is delayed for every setup, even though it might already be settled.

Why not going back to a low `interval` and moving the `peer.Settle` below the `peer.Join` logic?

Reply:
I would also expect `peer.Settle()` to be called after `peer.Join()`.

Reply:
moved, re-changed the interval value